lib/osf-components/addon/components/files/browse/component.ts

```diff
@@ -31,31 +29,16 @@ export default class FileBrowser extends Component {
         return toLeft;
     }
 
-    @computed('filesManager.{hasMore,loadingFolderItems}')
-    get shouldShowLoadMoreButton() {
-        return this.filesManager.hasMore && !this.filesManager.loadingFolderItems;
+    @computed('filesManager.currentFolder')
+    get notInRootFolder() {
+        const { currentFolder } = this.filesManager;
+        return currentFolder && Boolean(currentFolder.belongsTo('parentFolder').id());
     }
 
-    @computed('filesManager.displayedItems.[]', 'sort')
-    get sortedItems() {
-        let sortedItems = this.filesManager.displayedItems || [];
-
-        if (this.sort) {
-            const regex = /^(-?)([-\w]+)/;
-            const groups = regex.exec(this.sort)!;
-
-            groups.shift();
-            const [reverse, sortKey] = groups.slice(0, 2);
-
-            sortedItems = A(sortedItems).sortBy(sortKey);
-
-            if (reverse) {
-                sortedItems = sortedItems.reverse();
-            }
-        }
-
-        const folders = sortedItems.filterBy('kind', 'folder');
-
-        return [...folders, ...sortedItems.filter(item => item.kind !== 'folder')];
+    @computed('filesManager.{hasMore,loadingFolderItems,sortingFolderItems}')
+    get shouldShowLoadMoreButton() {
+        return this.filesManager.hasMore &&
+            !this.filesManager.loadingFolderItems &&
+            !this.filesManager.sortingFolderItems;
```

Comment:
seems like there's not much benefit to these as separate flags -- just an `isLoading` would do